packages/ckeditor5-list/docs/features/lists-editing.md

```diff
@@ -18,6 +18,12 @@ To edit a block inside a list item, press <kbd>Enter</kbd> to create a new line
 
 {@img assets/img/adding-list-items.gif 860 Editing a block list item.}
 
+## Multi-level lists
```

Comment:
It works the same in the standard list AFAICS.

Reply:
In commit.